source/assets/public/index.css

```diff
@@ -274,26 +293,10 @@ body {
   margin-top: -1rem;
 }
 
-
-
-
 .footer {
   margin-bottom: 6rem;
-  width: 90%;
```

Comment:
I think initially it was to better centre it in relevance to the left-aligned paragraph but it doesn't need to be there.